testdata/dnn/yolov4.cfg.readme

```diff
@@ -0,0 +1,2 @@
+[Origin](https://github.com/AlexeyAB/darknet/blob/c0d6b81a78e204c04c9bf4277974e0dadad0c4e2/cfg/yolov4.cfg)
+[License](https://github.com/AlexeyAB/darknet/blob/c0d6b81a78e204c04c9bf4277974e0dadad0c4e2/LICENSE)
```

Comment:
Can we add it to `yolov4.cfg` as comments?
```
# Origin: https://github.com/AlexeyAB/darknet/blob/c0d6b81a78e204c04c9bf4277974e0dadad0c4e2/cfg/yolov4.cfg
# License: https://github.com/AlexeyAB/darknet/blob/c0d6b81a78e204c04c9bf4277974e0dadad0c4e2/LICENSE
```

Reply:
I prefer to keep external files without extra modifications.

Reply:
Maybe we can just download it as weights?

Reply:
It is better to keep it `opencv_extra` (like others):
- there is no version control on share for downloaded files (they should not conflict at any time)
- configuration file is small and text based
- we may want to add test for importer without weights in the future (without requirement of huge model weights)